storage/zynq7000-flash/flashcfg.h

```diff
@@ -96,18 +96,22 @@ typedef struct {
 	uint8_t opCode;
 	uint8_t size;
 	uint8_t dummyCyc;
+	uint8_t dataLines;
 } flash_cmd_t;
 
 
-typedef struct {
+typedef struct flash_info {
 	flash_cfi_t cfi;
 	flash_cmd_t cmds[flash_cmd_end];
 
 	int readCmd; /* Default read command define for specific flash memory */
 	int ppCmd;   /* Default page program command define for specific flash memory */
 
 	const char *name;
-	enum { flash_3byteAddr, flash_4byteAddr } addrMode;
+	enum { flash_3byteAddr,
+		flash_4byteAddr } addrMode;
```

Comment:
clang-format is mistaken
```suggestion
	enum { flash_3byteAddr, flash_4byteAddr } addrMode;
```